apps/files/src/components/FileEntry.vue

```diff
@@ -83,7 +83,7 @@
 			class="files-list__row-mtime"
 			data-cy-files-list-row-mtime
 			@click="openDetailsIfAvailable">
-			<span>{{ mtime }}</span>
+			<NcDateTime :timestamp="this.source.mtime" :ignore-seconds="true" />
```

Comment:
```suggestion
			<NcDateTime :timestamp="source.mtime" :ignore-seconds="true" />
```
`this` is not needed in template

Reply:
changing this right now, thank you!